src/kbmod/search/layered_image.cpp

```diff
@@ -108,12 +106,12 @@ void LayeredImage::apply_mask_threshold(float thresh) {
     }
 }
 
-void LayeredImage::subtract_template(const RawImage& sub_template) {
+void LayeredImage::subtract_template(RawImage& sub_template) {
```

Comment:
Note for the future: Do we use this anywhere? Do we want to keep it? It seems like this should all be done in the standardizer before KBMOD.

Reply:
I really don't know. My preference would be to remove it too I guess, but it might be more challenging if we keep the array as a private member because we can't say something like `rawimg.image -= template` since the reference is const.